packages/edit-post/src/store/effects.js

```diff
@@ -72,6 +72,13 @@ const effects = {
 		} );
 	},
 	REQUEST_META_BOX_UPDATES( action, store ) {
+		// Saves the wp_editor fields
+		if ( window.tinymce ) {
```

Comment:
It will have the same effect but I'd use `import tinymce from "tinymce"` for consistency in how we use the tinymce module accross gutenberg.

Reply:
I'm not sure if this is the same. That will hold all the editor instances of all the rich text instances. `window.tinymce` holds the instances that were created by plugins and the classic block. Should this filter out classic block instances?

Reply:
E.g. in the classic block we set a property to `editor` and then check for it here.

Reply:
I updated the PR to import tinymce and use that vs the object stored in window.tinymce. This does have the effect of including the classic editor tinymce instances in .editors. Calling the save method on the classic block tinymce instances doesn't seem to have any adverse effects though (besides a minor performance hit). IMO it's not worth filtering them out.

Reply:
Going crazy here. I did a quick test of swapping back triggerSave vs the looping and it's working again. Going to do some more tests with the old PR to really figure out why that was failing for me sometimes again.

I'm afk for a couple hours, but will post an update soon. :D

Reply:
What I'm saying is that we _should_ use `window.tinymce` here.

Reply:
@iseulde Why? It should be the same thing as we use externals for "tinymce". using an import just makes the package more "npm" ready.

Reply:
Because the meta boxes were added with `window.tinymce`? The nom module is only used for RichText instances. We don't want to include them here. `window.tinymce.editors` does not include the  RichText TinyMCE instances.

Reply:
FYI my madness is that tinyMCE.triggerSave() --caps MCE-- works. The alternative PR had just tinymce.triggerSave() --lower mce-- which doesn't work.

I think using window.tinymce is safer with regards to the fix here. I feel like tinyMCE.editors was not always returning all of the added wp_editor boxes in some cases. Not sure the conditions.

Reply:
`import tinymce from "tinymce"` and `window.tinymce` are strictly equivalent according to our webpack config.

Reply:
FWIW, here is the code in core WordPress that makes sure we don't run save on editors on the text tab: https://github.com/WordPress/WordPress/blob/master/wp-includes/js/tinymce/plugins/wordpress/plugin.js#L605-L609

Reply:
You're right. I didn't see any rich text editor instances, so I thought they were different. Apparently rich text TinyMCE are not being added. Strange.